content/post/2024-10-01-same-same-but-different-how-advanced-data-science-techniques-help-us-validate-drug-names/index.md

```diff
@@ -0,0 +1,628 @@
+
+
+# Case Study: Validation of Drug Names for a Pharmaceutical Gig
+
+## Overview of the Project and its Objectives
+
+We did some detailed validation of eCRF data for one of our company's clients. This included things like demographics, lab results, and even concomitant medications (con-meds). 
+
+> Concomitant drugs refer to the simultaneous or successive use of two or more drugs in order to achieve the purpose of treatment, and the main result is to enhance the drug efficacy or to decrease the drug side effects. (ScienceDirect, https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/concomitant-drug)
+
+In our eCRF system, the researchers entered the drug names by hand. Unfortunately, text and humans don't always work well together. As you probably know, the popular acetylsalicylic acid can be written in a few different ways, including aspirin, salicylic acid, ASA, and... 2-acetoxybenzoic acid.
+
+Have you already figured out what problems this can cause? Computers are pretty simple — to them, "aspirin" and "Aspirin" are two different text strings. So, when we're counting the proportions of patients taking different drugs at different stages of treatment, we'll get two separate proportions for aspirin and Aspirin, even though they're the same drugs.
+
+So what solutions do we have?
+- See what unique drugs [`unique(con_meds)`] our database contains, **find errors and just correct them**.
+- See what unique drugs [`unique(con_meds)`] our database contains, **calculate the distance between their names, and then visualize them**.
+
+If our database has a relatively small number of con-meds, the first option is probably the easiest. However, the more drugs patients are taking, the more complex it becomes. This is because different drugs can have similar names, and sometimes patients have different characteristics that affect their drug regimen. To quickly and effectively identify duplicates, ambiguities, and errors in drug names, we need a more efficient method. **This is where Levenshtein  and t-SNE come in handy.**
```

Comment:
A lot of the content here seems to repeat points already covered earlier in the article. For example:
- The challenges with inconsistent drug names (e.g., “aspirin” vs. “Aspirin”) are already explained in detail in the ‘Introduction’ section.
- The concept of using distance metrics like Levenshtein and visualization methods like t-SNE is also well-covered in the sections on ‘t-SNE + Distance Metrics’ and ‘Clustering’.

Because these ideas have already been introduced and explained clearly, this section feels redundant. Streamlining by removing it could help keep the reader focused and improve the article’s flow. What do you think?